### ISSUE TITLE
Replace PHP library used by LTI provider to one compatible with LTI 1.3

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-lti-provider#64]
Swap current PHP library: https://github.com/Izumi-kun/LTI-Tool-Provider-Library-PHP with official LTI 1.3 PHP library: https://github.com/IMSGlobal/lti-1-3-php-library. Find/fix broken things that result.

**Comment by SteelWagstaff** [comment]
Other useful links: 
Sample WordPress multisite implementation: https://github.com/3iPunt/wordpress-lti-1-3.
Full Ruby implementation of platform & tool: https://lti-ri.imsglobal.org/.

**Comment by cagp-dev-mtl** [comment]
Installed locally the LTI tool example for understanding better how the plugin works:

https://github.com/IMSGlobal/lti-1-3-php-example-tool

**Comment by cagp-dev-mtl** [comment]
I have created a test LTI platform & tool for pressbooks on:

**Platform:**
https://lti-ri.imsglobal.org/platforms/688
application id: pb-lti-test

**Tool:**
https://lti-ri.imsglobal.org/lti/tools/692

***Setup steps:***
https://lti-ri.imsglobal.org/

**Comment by cagp-dev-mtl** [comment]
Update: I managed to create a connection between a local installation of Moodle and the imsgloba Testing tool I have created on IMSGlobal, next step is to be able to send requests back to Moddle from the tool.

The LTI 1.3 library and implementation have changed the way the platforms and tools communicate, after reviewing the code in the new 1.3 LTI plugin, we cannot just replace the library, many changes are required, starting from the config and login setups, I think I was very optimistic when estimating this ticket. 

I am still doing research on the full login spec for LTI 1.3 based on the example tool I have created and the local moodle installation. First I need to understand the new authentication in order for me to start working with the new LTI library. Unfortunately three other tickets of this sprint depends on this one and I believe I will be spending the next couple of days trying to make authentication work.

@SteelWagstaff , when you come back, I would like us to schedule some sessions for me to show ou the work and research I have done on the topic, I believe you have much more context and experience with the setup that will be very helpful for me to move on quicker.
